frontend/src/components/pages/Dashboard/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button, Container, Text } from "@chakra-ui/react";
+import React, { useContext, useState } from "react";
+import { Redirect, useHistory } from "react-router-dom";
+
+import DonorAPIClient from "../../../APIClients/DonorAPIClient";
+import SchedulingAPIClient from "../../../APIClients/SchedulingAPIClient";
+import * as Routes from "../../../constants/Routes";
+import AuthContext from "../../../contexts/AuthContext";
+import { Schedule } from "../../../types/SchedulingTypes";
+import DropoffCard from "./components/DropoffCard";
+
+const Dashboard = (): JSX.Element => {
+  const { authenticatedUser } = useContext(AuthContext);
+  const [schedule, setSchedule] = useState<Schedule[] | null>([]);
```

Comment:
why not name it as `[schedules, setSchedules]`, `schedule` makes it look like there is only one schedule.